haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc

```diff
@@ -90,61 +90,61 @@ function haskala_general_menu_default_menu_links() {
     'weight' => 0,
     'customized' => 1,
   );
-  // Exported menu link: menu-icons-menu_digital-books:<front>
-  $menu_links['menu-icons-menu_digital-books:<front>'] = array(
+  // Exported menu link: menu-icons-menu_digital-books:node/2592
```

Comment:
It's only for prevent link be active. that linkes's pages not exit yet.

On Tue, Jan 20, 2015 at 10:42 AM, Itamar Shapira Bar-Lev <
notifications@github.com> wrote:

> In
> haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc
> <https://github.com/Gizra/haskala/pull/53#discussion-diff-23208617>:
>
> > @@ -90,61 +90,61 @@ function haskala_general_menu_default_menu_links() {
> >      'weight' => 0,
> >      'customized' => 1,
> >    );
> > -  // Exported menu link: menu-icons-menu_digital-books:<front>
> > -  $menu_links['menu-icons-menu_digital-books:<front>'] = array(
> > +  // Exported menu link: menu-icons-menu_digital-books:node/2592
>
> this seems wrong - why do we mention this node id?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Gizra/haskala/pull/53/files#r23208617>.
>